packages/documentation/src/stories/components/buttons/close-button/close-button.docs.mdx

```diff
@@ -0,0 +1,34 @@
+import { Canvas, Controls, Meta } from '@storybook/blocks';
+import * as CloseButtonStories from './close-button.stories';
+
+<Meta of={CloseButtonStories} />
+
+<div className="docs-title">
+  # Close button
+
+  <nav>
+    <link-design of={JSON.stringify(CloseButtonStories)}></link-design>
+  </nav>
+</div>
+
+<div className="lead">
+  The close button is a web component. It is a slightly adapted version of the icon button, intended
+  for a specific use case.
+</div>
+
+<Canvas sourceState="shown" of={CloseButtonStories.Default} />
+<div className="hide-col-default">
+  <Controls of={CloseButtonStories.Default} />
+</div>
+
+## Usage in other components
+
+The `<post-closebutton>` component has a slot attribute itself (`<post-closebutton slot="post-closebutton">`) so that it can be easily added to other components without having to specify a slot name on the component when used.
+
+Definition of a component needing close button web component:
+
+<Canvas sourceState="shown" of={CloseButtonStories.SlotComponent} />
```

Comment:
This could be a simple code snippet. No need for a story here.
Use the storybook block `<Source/>` to implement a code snippet defined in a *.sample.html` file.

See for example the carousel component to learn how to do this.